plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/conf/hydra/sweeper/ax.yaml

```diff
@@ -0,0 +1,38 @@
+hydra:
+  # See https://ax.dev/
+  ax:
+    max_trials: 100
+    early_stop:
+      # Number of epochs without a significant improvement from
+      # the currently known best parameters
+      # An Epoch is defined as a batch of trials executed in parallel
+      max_epochs_without_improvement: 10
+      # An improvement larger than epsilon is considered significant
+      epsilon: 0.00001
+
+      minimize: ${hydra.ax.experiment.minimize}
+
+    experiment:
+      # Experiment name
+      name: ${hydra.job.name}
+      # Name of metric to optimize or null if you only return one metric
+      objective_name: objective
+      # Minimize objective:
+      #   true: minimize
+      #   false: maximize
+      minimize: false
+      # For the remaining paramaters, refer the Ax documentation: https://ax.dev/api/core.html#experiment
+      parameter_constraints: null
+      outcome_constraints: null
+      status_quo: null
+
+  sweeper:
+    class: hydra_plugins.hydra_ax_sweeper.AxSweeper
+    params:
+      experiment: ${hydra.ax.experiment}
+      early_stop: ${hydra.ax.early_stop}
+      verbose_logging: false
+      max_trials: ${hydra.ax.max_trials}
+      # set random seed here to make Ax results reproducible
+      random_seed: null
+      ax_params: ${hydra.ax.params}
```

Comment:
```yaml
  sweeper:
    class: hydra_plugins.hydra_ax_sweeper.AxSweeper
    params:
      ax:  ${hydra.ax}
```

You can simplify this by passing hydra.ax as is.
the code will need to be changed to accept the cfg object instead of the individual values.

This is better because the config is much more user visible.

Reply:
Not sure I follow what is missing here.

Reply:
This is not about missing, but about style.
Instead of manually unpacking ${hydra.ax} into params, you can just pass it directly per my example above.

Reply:
Got it :)

Reply:
Fixed via https://github.com/facebookresearch/hydra/pull/434/commits/f385339a2228783ac59ff89a1c5de574b3fcc242